### ISSUE TITLE
Japanese strategy doesn't show suggestions in niche cases

### ISSUE THREAD

**Issue by tadashi-aikawa** [tadashi-aikawa/obsidian-various-complements-plugin#77]
OK

![image](https://user-images.githubusercontent.com/9500018/152642196-78985fc6-779b-4fcb-9002-887daacd7872.png)

NG

![image](https://user-images.githubusercontent.com/9500018/152642199-a9066dbd-3be9-4ae9-add4-de6a3fe6377f.png)

```
[onTrigger] tokens is #,#,@smile
AutoCompleteSuggest.ts:657 [onTrigger] currentTokens is [{"word":"# @smile","offset":1},{"word":" @smile","offset":2},{"word":"smile","offset":4}]
AutoCompleteSuggest.ts:657 [onTrigger] currentToken is # @smile
```

`# @smile` is not expected.

**Comment by tadashi-aikawa** [comment]
`# @smi`
```
["#", " ", "@", "smi"]
```

`## @smi`
```
 ["#", "#", " @", "smi"]
```

`### @smi`
```
["#", "#", "# ", "@", "smi"]
```

`#### @smi`
```
["#", "#", "##", " ", "@", "smi"]
```

**Comment by tadashi-aikawa** [comment]
Fixed.

```
      .flatMap((x: string) =>
        x === " " ? x : x.split(" ").map((t) => (t === "" ? " " : t))
      );
```

**Comment by tadashi-aikawa** [comment]
Released at v5.1.0 🚀 

https://github.com/tadashi-aikawa/obsidian-various-complements-plugin/releases/tag/5.1.0
